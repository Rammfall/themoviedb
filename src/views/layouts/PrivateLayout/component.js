import React from 'react'
import PropTypes from 'prop-types'
import { Layout } from 'antd'

import Header from 'Views/components/stubs/Header'

const PrivateLayout = ({ children }) => (
  <Layout>
    <Header />
    <Layout.Content>{children}</Layout.Content>
  </Layout>
)

PrivateLayout.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.elementType),
    PropTypes.elementType
  ]).isRequired
}

export default PrivateLayout
