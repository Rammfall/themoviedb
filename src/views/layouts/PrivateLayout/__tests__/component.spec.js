import React from 'react'
import { shallow } from 'enzyme'

import PrivateLayout from '../component'

describe('PrivateLayout', () => {
  it('matches snapshot', () => {
    const component = shallow(<PrivateLayout>Test</PrivateLayout>)

    expect(component).toMatchSnapshot()
  })
})
