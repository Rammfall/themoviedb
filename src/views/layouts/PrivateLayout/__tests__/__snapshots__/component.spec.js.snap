// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivateLayout matches snapshot 1`] = `
<Layout>
  <Header />
  <Content>
    Test
  </Content>
</Layout>
`;
